_includes/v2.6/release-notes/v2.6.9-release-notes.md

```diff
@@ -0,0 +1,9 @@
+18 April 2018
+
+#### Bug fixes
+
+- Fix an interaction between failsafe inbound/outbound ports and do-not-track policy that resulted in failsafe ports being blocked if do-not-track policy was added. [felix #1748](https://github.com/projectcalico/felix/pull/1748) (@fasaxc)
+
+- Fix a bug where the Egress type was displayed for upgraded pre-dnat network policies [libcalico-go #851](https://github.com/projectcalico/libcalico-go/pull/851) (@tmjd)
```

Comment:
Nit: lowercase egress....at the same time dnat should be all uppercase, as an acronym